<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPUInfo</class>
 <widget class="QWidget" name="CPUInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_bg">
     <property name="styleSheet">
      <string notr="true">#frame_bg
{
	border-radius: 10px;
	background-color: rgba(0, 0, 0, 50%);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>320</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>320</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#frame
{
	background: #152034;
	border-radius: 10px;
}

QLabel
{
	font-size: 16px;
	font-family: 微软雅黑;
	color: #01EEC3;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>16</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>8</number>
         </property>
         <property name="rightMargin">
          <number>8</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item alignment="Qt::AlignRight">
          <widget class="QPushButton" name="pushButton">
           <property name="minimumSize">
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	border-image: url(:/images/images/ic_gb.png);
}

QPushButton:hover
{
	border-image: url(:/images/images/ic_gb_bright.png);
}</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_pageSize"/>
         </item>
         <item>
          <widget class="QLabel" name="label_minAddress"/>
         </item>
         <item>
          <widget class="QLabel" name="label_maxAddress"/>
         </item>
         <item>
          <widget class="QLabel" name="label_mask"/>
         </item>
         <item>
          <widget class="QLabel" name="label_processorNum"/>
         </item>
         <item>
          <widget class="QLabel" name="label_processorType"/>
         </item>
         <item>
          <widget class="QLabel" name="label_processorLevel"/>
         </item>
         <item>
          <widget class="QLabel" name="label_processorVersion"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="GetPCInfo.qrc"/>
 </resources>
 <connections/>
</ui>
